test/test-utils/matchers/to-have-weather-matcher.ts

```diff
@@ -2,6 +2,7 @@ import { capitalizeString } from "#app/utils/string-utils";
 import { WeatherType } from "#enums/weather-type";
 import { isGameManagerInstance, receivedStr } from "#test/test-utils/testUtils";
 import type { MatcherState, SyncExpectationResult } from "@vitest/expect";
+import { isNullOrUndefined } from "util";
```

Comment:
```suggestion
import { isNullOrUndefined } from "#app/utils/common";
```
You might need to merge in latest beta for this to be the correct import location.

Reply:
```suggestion
import { isNullOrUndefined } from "#app/utils/common-utils";
```
FTFY

Reply:
Ah right, thanks.